File: trunk/src/com/ff/ui/form/Label.java
package com.ff.ui.form;

import com.ff.Component;
import com.ff.annotation.Property;

public class Label extends Component{

	@Property
	private String text;

	public String getText() {
		return text;
	}

	public void setText(String text) {
		this.text = text;
	}
	
	
}

File: zkext/trunk/src/web/js/zkext/ui/form/Label.js
/* Label.js*/

zkext.ui.form.Label = zk.$extends(zkext.ui.Component,{	 
	$define: {	 
		text:function(val){
			this.setProperty("text","setText",val);
		}
	},
	
	configure_:function(){
		this.$supers('configure_');		
		 		
	 
	},
	createExt_:function(){
		this.newInstance('Ext.form.Label');
	} 	
});